codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/ServiceRuntimePluginGenerator.kt

```diff
@@ -21,6 +21,32 @@ import software.amazon.smithy.rust.codegen.core.smithy.customize.writeCustomizat
 import software.amazon.smithy.rust.codegen.core.util.dq
 
 sealed class ServiceRuntimePluginSection(name: String) : Section(name) {
+    /**
+     * Hook for adding HTTP auth schemes.
+     *
+     * Should emit code that looks like the following:
+     * ```
+     * .auth_scheme("name", path::to::MyAuthScheme::new())
+     * ```
+     */
+    data class HttpAuthScheme(val configBagName: String) : ServiceRuntimePluginSection("HttpAuthScheme")
+
+    /**
+     * Hook for adding retry classifiers to an operation's `RetryClassifiers` bundle.
+     *
+     * Should emit code that looks like the following:
+     ```
+     .with_classifier(AwsErrorCodeClassifier::new())
+     */
+    data class RetryClassifier(val configBagName: String) : ServiceRuntimePluginSection("RetryClassifier")
```

Comment:
These were recently removed in main. Seems like this was a merge error?

Reply:
oops

Reply:
removed